packages/frontend/src/components/UserSettings/SlackSettingsPanel/index.tsx

```diff
@@ -107,77 +126,135 @@ const SlackSettingsPanel: FC = () => {
+                            <Stack spacing="sm">
+                                <TextInput
+                                    label="Enter the URL of a profile photo for your Slack App"
+                                    size="xs"
+                                    placeholder="https://lightdash.cloud/photo.jpg"
+                                    type="url"
+                                    disabled={!isValidSlack}
+                                    {...form.getInputProps(
+                                        'appProfilePhotoUrl',
+                                    )}
+                                    value={
+                                        form.values.appProfilePhotoUrl ??
+                                        undefined
+                                    }
+                                />
+                                <Stack justify="center">
+                                    <Title order={6}>Profile photo</Title>
+                                    <Avatar
+                                        src={form.values?.appProfilePhotoUrl}
+                                        size="xl"
+                                        radius="md"
+                                        bg="gray.1"
+                                    />
+                                </Stack>
+                                <Select
+                                    label={
+                                        <Group spacing="two" mb="two">
+                                            <Text>
+                                                Select a notification channel
+                                            </Text>
+                                            <Tooltip
+                                                multiline
+                                                maw={250}
+                                                label="Choose a channel where to send notifications to every time a scheduled delivery fails. You have to add this Slack App to this channel to enable notifications"
+                                            >
+                                                <MantineIcon
+                                                    icon={IconHelpCircle}
+                                                />
+                                            </Tooltip>
+                                        </Group>
+                                    }
+                                    disabled={isLoadingSlackChannels}
+                                    size="xs"
+                                    placeholder="Select a channel"
+                                    searchable
+                                    clearable
+                                    nothingFound="No channels found"
+                                    defaultValue={
+                                        form.values.notificationChannel
+                                    }
```

Comment:
I don't think this will be necessary after the change mentioned above. im pretty sure this line throws a warning too